docs/data-sources/aviatrix_network_domains.md

```diff
@@ -0,0 +1,38 @@
+---
+subcategory: "TGW Orchestrator"
+layout: "aviatrix"
+page_title: "Aviatrix: aviatrix_network_domains"
+description: |-
+  Get a list of all Network Domains and detail
```

Comment:
Could be worded better. `Gets a list of all Network Domains.`

Reply:
have fixed